avm/res/compute/virtual-machine/extension/main.json

```diff
@@ -0,0 +1,160 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "languageVersion": "2.0",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.23.1.45101",
```

Comment:
I wouldn't have asked normally, but since bicep 0.24 is out and we're regenerating all readmes with that version as part of another PR, could you please rerun set-avmmodule with the latest bicep version?

Reply:
done